first_ruby/config/application.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "boot"
+
+require "rails/all"
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module FirstRuby
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 6.1
+
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
+    #
+    # config.time_zone = "Central Time (US & Canada)"
```

Comment:
I see this is commented. Since this is commented, what timezone will this app runs on?

Reply:
It will run on whatever timezone your Operating System is on.

Reply:
So, if this app lives on a server in US (EST), what timezone will it be?

Reply:
It will follow the timezone according to the server. For example, Heroku has default timezone of UTC.